submarine-cloud/manifests/submarine-cluster/service.yaml

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+kind: Service
+apiVersion: v1
+metadata:
+  name: submarine-svc
+spec:
+  selector:
+    app: cluster-test
+  ports:
+    - port: 8080
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: submarine-ingress
+  annotations:
+    ingress.kubernetes.io/rewrite-target: /
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /
+            backend:
+              serviceName: submarine-svc
+              servicePort: 8080
+
+---
```

Comment:
Is this comment for the submarine-ingress? How about we put the comments before submarine-ingress definition.

Reply:
Only after it is executed can you follow this command, so it is placed behind.
After completing the development of the submarine operator, will write detailed documentation.